drivers/display/display_stm32_ltdc.c

```diff
@@ -57,13 +57,13 @@ LOG_MODULE_REGISTER(display_stm32_ltdc, CONFIG_DISPLAY_LOG_LEVEL);
 #if defined(CONFIG_HAS_CMSIS_CORE_M)
 #include <cmsis_core.h>
 
-#if __DCACHE_PRESENT == 1
+#if defined(CONFIG_DCACHE)
```

Comment:
Informational: Could use zephyr cache API here. See https://github.com/zephyrproject-rtos/zephyr/issues/67302.

Reply:
Sure, not requested for this PR.